### DISCUSSION TITLE
[Request] Only showing available ranks.

### DISCUSSION THREAD

**Discussion by Eternatum** [Zrips/CMI#1255] [Enhancement]
I would like to be able to only show available ranks to players. I do not them to see the full list of ranks available to them. Is this already possible? If not, it would be a nice addition.
---
CMI Version 8.3.1.0
Server Type: Paper 1.13.2
Server Version: git-Paper-480 (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)

**Comment by Zrips** [comment]
Can you define "available ranks"? Like only next ranks player can rankup into? Should we show previous ranks from the list? 
Extra information is always appreciated on topics like this.

**Comment by Eternatum** [comment]
> Can you define "available ranks"? Like only next ranks player can rankup into? Should we show previous ranks from the list?
> Extra information is always appreciated on topics like this.

Meaning if I belong to group a, and have rank a1; I cannot see the available ranks in group b. So basically, all I can see is my current path and the ranks in that path.
